en/docs/develop-components/work-with-the-micro-integrator-runtime-in-choreo.md

```diff
@@ -61,10 +61,83 @@ Here are some of the benefits of adding third-party JARs to the Micro Integrator
  - Optimized performance: By offering optimized implementations of common functions, third-party libraries can improve the performance of the Micro Integrator.
  - Efficient development: Developers can avoid re-implementing commonly used functions, reducing development time and increasing efficiency.
 
-To incorporate third-party libraries into your Micro Integrator project, create a new directory called `libs` at the root of your project. If your project root and GitHub repository root are not the same, place the `libs` directory in the project root. After adding the required JAR files to the `libs` directory, the Micro Integrator runtime will automatically include them when deploying the component. This process ensures that any dependencies needed by your integration flow are available at runtime.
+To incorporate third-party libraries into your Micro Integrator project, create a new directory called `libs` at the
+root of your project. If your project root and GitHub repository root are not the same, place the `libs` directory in
+the project root. After adding the required JAR files to the `libs` directory, the Micro Integrator runtime will
+automatically include them when deploying the component. This process ensures that any dependencies needed by your
+integration flow are available at runtime.
+
+In addition to `jar` libraries, you may also need to incorporate OSGi bundle JARs into your project. To do this,
+create another directory called `dropins` at the root of your project. Similarly, if the project root and GitHub
+repository root differ, place the `dropins` directory in the project root.
+
+Within the `dropins` directory, add the OSGi bundle JAR files. These bundles will be picked up by the OSGi runtime during
```

Comment:
Let's not make this a separate paragraph, and have it in the same para.

Reply:
These bundles will be picked up by the OSGi runtime during -> The OSGI runtime will pick up these bundles

Reply:
you mean
```
The OSGI runtime will pick up these bundles, enabling you to utilize their
functionality within your Micro Integrator project.
```

Reply:
The OSGI runtime will pick up these bundles during deployment, enabling you to utilize their functionality within your Micro Integrator project.